modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc

```diff
@@ -438,3 +438,22 @@ PostgreSQL: \c owncloud; \d
 MySQL     : quit
 PostgreSQL: \q
 ....
+
+[[how-to-solve-deadlock-error]]
+=== How to solve deadlock error
```

Comment:
Please use [title case](https://www.snappysentences.com/sentence-case-v-title-case/) for headers.

Reply:
why not just write "replace this with this"? 

If you have a question for me I don't tell you "Yeah mate, just google this and then you get a page, read that page, somewhere on that page you get the answer I could tell you right now"